control/server.py

```diff
@@ -72,15 +94,11 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_value, traceback):
         """Cleans up SPDK and server instances."""
+        if exc_type is not None:
+            self.logger.error("GatewayServer exception occurred:", exc_info=(exc_type, exc_value, traceback))
```

Comment:
nit: (here and in all other cases) I would like we keep the 80 characters line limit, breaking longer lines where it is possible.

Reply:
BTW, `logging` already provides a logger method `.exception()` which equals to `.error()` plus traceback dump.